.github/workflows/bump-versions.yml

```diff
@@ -28,12 +39,24 @@ jobs:
         working-directory: ./src
         run: go run ./cmd/bump-versions
 
+      - name: Get branch
+        id: get-branch
+        run: |
+          if [ "${{ inputs.environment }}" = "Production" ]; then
```

Comment:
You are passing `inputs.environment` as the `name` environment variable above. You could use `${name}` instead of the GitHub expression here. I believe, it is generally preferred to pass in inputs as environment variables to avoid injection issues, but it doesn't matter in this case.

Reply:
Yes, the `environment` GH feature is quite confusing.